cypress/integration/page.spec.js

```diff
@@ -861,7 +920,9 @@
       describe('', () => {
         beforeEach(() => {
           cy.clock();
-          page.spyOn('**/comments?postId=2', 'post2Coments', { fixture: 'post2Comments' });
+          page.spyOn('**/comments?postId=2', 'post2Coments', {
```

Comment:
There seems to be a typo in the alias name 'post2Coments'. It should likely be 'post2Comments'.